VirtoCommerce.PricingModule.Data/Search/ProductPriceDocumentChangesProvider.cs

```diff
@@ -14,11 +15,12 @@ namespace VirtoCommerce.PricingModule.Data.Search
 {
     public class ProductPriceDocumentChangesProvider : IPricingDocumentChangesProvider
     {
+        private const string ChangeLogObjectType = nameof(PriceEntity);
```

Comment:
Private names should start with `_` and lowercase letter.

Reply:
Not private consts. They should start with uppercase.

Reply:
No, they shouldn't.

Reply:
Well, private `const`s and `static readonly` fields are not covered by [Framework design guidelines by Microsoft](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines), nor by the [RSDN guidelines](https://rsdn.org/article/mag/200401/codestyle.XML) (which is a de-facto standard). However, default ReSharper rules suggest to name constants and static readonly fields using the `PascalCase`.

Reply:
Here is my point:
When you read the code and see `_changeLogObjectType` you understand that this is something private and not visible outside current class, and actually you don't care whether this is a field or a constant.